src/functions-and-arrays.js

```diff
@@ -52,14 +91,31 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+//function uniquifyArray()
+function uniquifyArray(words) {
```

Comment:
You were almost there on this one here!! Instead of having uniquifyArray(wordsUnique) as uniqueWords, you should've passed an []! And you also forgot to return null.

A solution based on yours:

```
function uniquifyArray(words) {
let uniqueWords = [];
  if (!words.length){
    return null;
  }
  for(let i = 0; i < words.length; i++) {
    if(uniqueWords.indexOf(words[i]) === -1){
      uniqueWords.push(words[i])
    }
  }
  return uniqueWords;
}
```